File: src/examples/4-divide/odd_numbers_bad.cs
string printOnlyOddNumbers(int[] numbers)
{
	List<int> n = new List<int>() { };
	List<int> rev = new List<int>() { };
	foreach (var number in numbers)
	{
		if (number % 2 == 1)
		{
			n.Add(number);
		}
	}
	for (int i = n.Count - 1; i >= 0; i--)
	{
		rev.Add(n.ElementAt(i));
	}
	StringBuilder sb = new StringBuilder();
	for (int i = 0; i < rev.Count; i++)
	{
		sb.Append(rev.ElementAt(i));
	}
	return sb.ToString();
}

File: src/examples/4-divide/odd_numbers_bad.ts
function printOnlyOddNumbers(numbers: number[]): string {
  let odds: number[];
  const rev: number[] = [];
  odds = numbers.filter(n => n % 2);
  for (let i = odds.length - 1; i >= 0; i--) {
    rev.push(numbers[i]);
  }
  let sb = '';
  for (let i = 0; i < rev.length; i++) {
    sb += rev[i];
  }
  return sb;
}
